apps/src/applab/ConnectedStudioAppWrapper.jsx

```diff
@@ -0,0 +1,19 @@
+/** @file Redux-connected version of StudioAppWrapper, used by App Lab. */
+// Strict linting: Absorb into global config when possible
+/* jshint
+ unused: true,
+ eqeqeq: true,
+ maxlen: 120
+ */
+'use strict';
+
+var connect = require('react-redux').connect;
+var StudioAppWrapper = require('../templates/StudioAppWrapper.jsx');
+
+module.exports = connect(
+  (state) => ({
+    assetUrl: state.assetUrl,
+    isEmbedView: state.isEmbedView,
+    isShareView: state.isShareView
+  })
+)(StudioAppWrapper);
```

Comment:
This might be fairly long-term.  I don't forsee a use for Redux in our "older" apps in the near future, and this happens to be the root of several components currently shared between them and App Lab.

I don't see having a separate 'connected' version as an issue, especially given that it's the recommended react-redux approach.  Do you expect this to be confusing?